src/classes/business_object.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC
+from uuid import UUID, uuid4
+from datetime import datetime
+
+
+class BusinessObject(ABC):
+
+    def __init__(self) -> None:
+        self._id = uuid4()
+        self._creation_date = datetime.now()
+
+    def get_id(self) -> UUID:
+        return self._id
+
+    def get_creation_date(self) -> datetime:
+        return self._creation_date
```

Comment:
Don't we also need setter functions for them?

Reply:
It isn't necessary since the creation date and the uuid is set when initialising the object, which inherits the business object. 
datetime.now() -> sets the current time 
uuid4() -> generates a new uuid of type 4
Neither the creation date nor the uuid of a instance needs to be changed or should be changed after that.